server/api/save/[id].post.ts

```
@@ -1,23 +1,264 @@
+    case BoardAccessLevel.LIMITED_EDIT:
+      // Anonymous users cannot edit LIMITED_EDIT boards
+      if (!profileId) return false;
+
+      // Authenticated users need EDITOR or OWNER role
+      const editorAccessRecord = await db.query.BOARD_ACCESS.findFirst({
+        where: and(
+          eq(BOARD_ACCESS.board_id, board.board_id),
+          eq(BOARD_ACCESS.profile_id, profileId),
+          sql`${BOARD_ACCESS.role} IN ('editor', 'owner')`,
+        ),
+      });
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)